docker-builds/reference-crucible/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM phusion/passenger-full:0.9.18
+MAINTAINER Erik Wiffin
+
+# Enable nginx and passenger
+RUN rm -f /etc/service/nginx/down
+
+### Our Stuff ###
+# Update Apt
+RUN sh -c "echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' >> /etc/apt/sources.list" && \
+    apt-key adv --keyserver keyserver.ubuntu.com --recv 7F0CEB10 && \
+    apt-get update
+
+# Install MongoDB
+RUN apt-get install -y mongodb mongodb-server && \
+    mkdir -p /data/db && \
+    service mongodb start
+
+# Make sure it runs
+RUN mkdir -p /etc/service/mongodb/
+ADD mongodb.sh /etc/service/mongodb/run
+
+# Install Bower
+RUN apt-get install -y nodejs && \
+    npm install bower -g
+
+# Remove default nginx site config
+RUN rm /etc/nginx/sites-enabled/default
+
+# Provide our own nginx site config
+ADD crucible.conf /etc/nginx/sites-enabled/crucible.conf
+
+# Switch to "app" user
+USER app
+
+# Clone crucible project
+RUN git clone https://github.com/fhir-crucible/crucible.git /home/app/crucible
```

Comment:
Question looking forward: if (for development purposes) we map a local host volume to this point, will ruby code "just work" (as we'd do in a development workflow for our nodejs components, say), or are additional steps required?